server/src/routes/auth.js

```diff
@@ -1,8 +1,14 @@
 const express = require('express');
 const passport = require('passport');
+const connection = require('../init/setupMySql');
 
 const router = express.Router();
 
+const checkIfUserAdmin = (req, res) => {
```

Comment:
What is this function for?

Reply:
This can be deleted. Just forgot to do it.